src/app/cli/commands.ts

```diff
@@ -618,6 +625,164 @@ const modelCommands: CommandDefinition = {
     },
 };
 
+/**
+ * MCP management commands
+ */
+const mcpCommands: CommandDefinition = {
+    name: 'mcp',
+    description: 'Manage MCP servers',
+    usage: '/mcp <subcommand> [args]',
+    category: 'MCP Management',
+    aliases: ['mc'],
+    subcommands: [
+        {
+            name: 'list',
+            description: 'List all available MCP servers',
+            usage: '/mcp list',
+            handler: async (_args: string[], agent: SaikiAgent) => {
+                try {
+                    const clients = agent.mcpManager.getClients();
+                    const failedConnections = agent.mcpManager.getFailedConnections();
+
+                    if (clients.size === 0 && Object.keys(failedConnections).length === 0) {
+                        console.log(chalk.yellow('📋 No MCP servers configured or connected.'));
+                        return true;
+                    }
+
+                    console.log(chalk.bold.blue('\n🔌 MCP Servers:\n'));
+                    for (const [name] of clients) {
+                        console.log(chalk.green(`✅ ${name}:`));
+                        console.log(chalk.dim(`  Connected: Yes`));
+                        console.log();
+                    }
+
+                    if (Object.keys(failedConnections).length > 0) {
+                        console.log(chalk.bold.red('\n❌ Failed Connections:\n'));
+                        for (const [name, error] of Object.entries(failedConnections)) {
+                            console.log(chalk.red(`❌ ${name}:`));
+                            console.log(chalk.dim(`  Error: ${error}`));
+                        }
+                    }
+
+                    console.log(
+                        chalk.dim(
+                            '💡 Use /mcp add <name> <config_json_string> to connect a new MCP server.'
+                        )
+                    );
+                    console.log(
+                        chalk.dim('💡 Use /mcp remove <name> to disconnect an MCP server.')
+                    );
+                    console.log(chalk.dim('💡 Use /mcp help for detailed command descriptions.'));
+                    return true;
+                } catch (error) {
+                    logger.error(
+                        `Failed to list MCP servers: ${error instanceof Error ? error.message : String(error)}`
+                    );
+                }
+                return true;
+            },
+        },
+        {
+            name: 'add',
+            description: 'Add a new MCP server',
+            usage: '/mcp add <name> <config_json_string>',
```

Comment:
can you add examples for each type of MCP server (stdio, http, SSE) and also show the testing details for this in screenshot?